### PR TITLE
fix(clerk-js): Finalize all text colors

### PR DESCRIPTION
**Pull request by anagstef**
## Description

Remove `colorTextTertiary` and `colorTextLabel`.

Use the following text colors:
- `colorText: #212126`
- `colorTextSecondary: #747686`

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by anagstef** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @anagstef, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-objwdx9b6.clerkstage.dev) | Feb 08, 2024 04:03 PM |

**Review by octoper — APPROVED** [review]

File: packages/clerk-js/src/ui/foundations/colors.ts [hunk 1]
```diff
@@ -37,23 +37,19 @@ export const neutralAlpha = Object.freeze({
...
   avatarBackground: neutralAlpha.neutralAlpha400,
   modalBackdrop: neutralAlpha.neutralAlpha700,
-  activeDeviceBackground: whiteAlpha.whiteAlpha200,
   // Themable colors
   ...neutralAlpha,
   ...whiteAlpha,
   colorBackground: 'white',
   colorInputBackground: 'white',
-  colorText: '#131316',
+  colorText: '#212126',
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/foundations/colors.ts:48]: Why are we changing this?

File: packages/clerk-js/src/ui/foundations/colors.ts [hunk 2]
```diff
@@ -37,23 +37,19 @@ export const neutralAlpha = Object.freeze({
...
   colorTextSecondary: '#747686',
-  colorTextTertiary: '#9394A1',
-  colorTextLabel: '#42434D',
   colorInputText: 'black',
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/foundations/colors.ts:50]: Are we sure this needs to be black ?

File: packages/types/src/appearance.ts [hunk 3]
```diff
@@ -426,79 +429,72 @@ export type Variables = {
   colorNeutral?: CssColorOrAlphaScale;
   /**
    * The default text color.
-   * @default #131316
+   * @default '#212126'
    */
   colorText?: CssColor;
   /**
    * The text color for elements of lower importance, eg: a subtitle text.
-   * @default A lighter shade of {@link Variables.colorText}
```
Comment (desiprisg) [inline packages/types/src/appearance.ts:434]: I think we need to keep this comment. The default changes if `colorText` changes.

File: packages/clerk-js/src/ui/primitives/Text.tsx [hunk 4]
```diff
@@ -23,7 +20,6 @@ const { applyVariants, filterProps } = createVariants(theme => {
         danger: { color: theme.colors.$danger500 },
         success: { color: theme.colors.$success500 },
         neutral: { color: theme.colors.$colorTextSecondary },
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/primitives/Text.tsx:22]: Can we have `primary` -> `body` and `neutral` -> `secondary` ? I think that would be cleaner.

File: packages/clerk-js/src/ui/foundations/colors.ts (hunk 1 above)
Comment (anagstef) [reply packages/clerk-js/src/ui/foundations/colors.ts:48]: That has been discussed with the design team and we concluded to use this color.

File: packages/clerk-js/src/ui/foundations/colors.ts (hunk 2 above)
Comment (anagstef) [reply packages/clerk-js/src/ui/foundations/colors.ts:50]: @desiprisg we can change it to `#212126` to match the `colorText` variable (?)

File: packages/clerk-js/src/ui/primitives/Text.tsx (hunk 4 above)
Comment (anagstef) [reply packages/clerk-js/src/ui/primitives/Text.tsx:22]: Good call! I will add a new item to refactor this later.

File: packages/types/src/appearance.ts (hunk 3 above)
Comment (anagstef) [reply packages/types/src/appearance.ts:434]: I think we should explicitly say the default here, and maybe move this comment up to the descriotion

**Pull request merged**